src/Phinx/Console/Command/Create.php

```diff
@@ -57,6 +58,8 @@ protected function configure(): void
 
         // Allow the migration path to be chosen non-interactively.
         $this->addOption('path', null, InputOption::VALUE_REQUIRED, 'Specify the path in which to create this migration');
+
+        $this->addOption('style', null, InputOption::VALUE_REQUIRED, 'Specify the style of migration to create');
```

Comment:
This creates a BC issue for https://github.com/cakephp/migrations/actions/runs/3583067226/jobs/6028049748 it seems for new release of supporting 0.13

Could we just default to one somewhere maybe? Not having it required?